server/data/interfaces/component.ts

```diff
@@ -0,0 +1,5 @@
+export default interface Component {
+  html: string
+  css?: string[]
+  javascript?: string[]
+}
```

Comment:
the ? can actually go now. Have changed so that they return empty arrays if there are none